### DISCUSSION TITLE
November|Community Manager Monthly Report

### DISCUSSION THREAD

**Discussion by Cindy884** [ringdao-community-guild/discussion#57] [Show and tell]
# Overview

November witnessed great growth in DCDAO. DCDAO membership grew to 12 people and we collaborated with Litnery, cryptomeda, and AIWEB3 on [[giveaway.com](http://giveaway.com/)](http://giveaway.com) which gained great exposure.
<img width="421" alt="image" src="https://github.com/dcdao/discussion/assets/109792098/4ecbe66a-9c79-4ad2-aa42-cf48116d674d">
# Details

### Thanksgiving Giveaway with the other three partners

Description: DCDAO November Giveaway in [giveaway.com](http://giveaway.com/)

LINK:https://x.com/Official_DCDAO/status/1731650056510488642?s=20

Data Review: 1000+ participants; 700+ twitter followers; 1000+ discord new members

### Medium Article Publish

There were three articles posted in the medium account.

****[[What is the difference between DCDAO, Darwinia Community, and Darwinia Github Organization?](https://medium.com/@realDCDAO2023/what-is-the-difference-between-dcdao-darwinia-community-and-darwinia-github-organization-1f442f35db50?source=your_stories_page-------------------------------------)]****

****[[How to protect your assets in Web 3.0?](https://medium.com/@realDCDAO2023/how-to-protect-your-assets-in-web3-0-c8f4eaf7464d?source=your_stories_page-------------------------------------)]****

****[[DCDAO Monthly News](https://medium.com/@realDCDAO2023/dcdao-monthly-news-566603da3c19?source=your_stories_page-------------------------------------)]****

## Community Management

DCDAO& Darwinia Quiz Game will be a regular event on the Discord server. 

11 members joined the event on average in November.

https://discord.gg/KwQHF6sv

### DCDAO’s Proposal to Darwinia Treasury (Q1 2024)

Link：https://darwinia2.subsquare.io/democracy/proposals/2

# December Plan
<img width="432" alt="image" src="https://github.com/dcdao/discussion/assets/109792098/211b244c-bb11-49e1-947b-d6a6b322dcce">

**Comment by hackfisher** [comment]
Well done, Cindy, looking forward more contributors to join DCDAO
